src/ffi/mod.rs

```diff
@@ -1063,8 +1689,70 @@ impl Subscriber for TelioTracingSubscriber {
     }
 }
 
+pub struct UniffiTelioTracingSubscriber {
```

Comment:
Why is this a copy pasted version of `TelioTracingSubscriber`?

Reply:
Because the old code is still alive, but the new code should still be tested. There will be a cleanup PR later where all of the duplication introduced here is removed (the `uniffi_` and `UniffI` prefixes will be removed at the same time)

Reply:
I think that if you change the `TelioTracingSubscriber` to also use `TelioLoggerCb` there will be no need for `UniffiTelioTracingSubscriber`.

Reply:
I added an enum to wrap the callback